CMakeLists.txt

```diff
@@ -254,7 +254,8 @@ set(BUILD_ARGS
         -DUSE_OPENSSL=${USE_OPENSSL}
         -DUSE_MBEDTLS=${USE_MBEDTLS}
         -DKVS_DEFAULT_STACK_SIZE=${KVS_DEFAULT_STACK_SIZE}
-        -DCMAKE_C_FLAGS=${CMAKE_C_FLAGS})
+        -DCMAKE_C_FLAGS=${CMAKE_C_FLAGS}
+        -DPKG_CONFIG_EXECUTABLE=${PKG_CONFIG_EXECUTABLE})
```

Comment:
We should not need to pass this, the build of the dependency should be searching for and finding pkg-config it shouldn't need to be told where it is.

Reply:
I fixed this in my current producer c PR to install pkgconfiglite for windows. So this should not be necessary.

Reply:
Thanks, I've tried it out

Reply:
Installing that package is not working for some reason: https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/actions/runs/8741051317/job/23986196094?pr=1965#step:6:7565